### PR TITLE
feat(datepicker): container option

### PR DESCRIPTION
**Pull request by mschoudry**
feature request #1380 
added support for container option

### CODE REVIEW

File: src/datepicker/datepicker-input.spec.ts [hunk 1]
```diff
@@ -522,6 +522,49 @@ describe('NgbInputDatepicker', () => {
...
+      // open date-picker
+      const button = fixture.nativeElement.querySelector('button');
+      button.click();
+      fixture.detectChanges();
+
+      expect(fixture.nativeElement.querySelector('ngb-datepicker')).toBeNull();
+      expect(document.querySelector(selector).querySelector('ngb-datepicker')).not.toBeNull();
+    });
+
+    it('should properly destroy typeahead window when the "container" option is used', () => {
```
Comment (sousarka) [inline src/datepicker/datepicker-input.spec.ts]: datepicker not "typeahead"

Comment (mschoudry) [reply src/datepicker/datepicker-input.spec.ts]: oopss... :)

File: src/datepicker/datepicker-input.ts [hunk 2]
```diff
@@ -130,6 +130,12 @@ export class NgbInputDatepicker implements OnChanges,
   @Input() startDate: {year: number, month: number};
 
   /**
+   * A selector specifying the element the tooltip should be appended to.
```
Comment (pkozlowski-opensource) [inline src/datepicker/datepicker-input.ts]: 'tooltip' => datepicker popup.

**Comment by pkozlowski-opensource** [comment]
@mschoudry thnx, this LGTM. Could you just please update the documentation based on the comment I've left in the code? Thnx!

**Pull request closed without merging**